### PR TITLE
Adds integration tests in the repo for customImportMap plugin

### PR DESCRIPTION
**Pull request by Shivamdhar**
### Description
- Adds Integration tests in the repo for `customImportMapDashboards` plugin
  - [x] Sanity check for presence of import vector map tab
  - [x] Verifies successful geoJSON file upload
### Issues Resolved
- Integrates with geospatial backend plugin for tests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/cypress-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          path: geospatial
+          repository: opensearch-project/geospatial
+          ref: '2.2'
+      - name: Run Opensearch with plugin
+        run: |
+          cd geospatial
+          ./gradlew run &
+          timeout 600 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:9200)" != "200" ]]; do sleep 5; done'
+      - name: Checkout Plugin
+        uses: actions/checkout@v2
```
Comment (CCongWang) [inline .github/workflows/cypress-workflow.yml:40]: I'm seeing the path is specified in another 2 workflows, not sure if it's needed:
ism: https://github.com/opensearch-project/index-management-dashboards-plugin/blob/main/.github/workflows/cypress-workflow.yml#L42-L43
alerting: https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/.github/workflows/cypress-workflow.yml#L46-L47

File: src/plugins/custom_import_map/cypress.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "defaultCommandTimeout": 10000,
+  "env": {
+    "opensearch_url": "localhost:9200",
+    "opensearch_dashboards": "http://localhost:5601",
+    "security_enabled": false,
+    "base_url": "http://localhost:5601",
```
Comment (CCongWang) [inline src/plugins/custom_import_map/cypress.json:7]: this is not needed

File: src/plugins/custom_import_map/cypress/integration/geojson_file_upload.spec.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+/// <reference types="cypress" />
+
+import { BASE_PATH } from '../utils/constants';
+import 'cypress-file-upload';
+
+describe('Verify successful custom geojson file upload', () => {
+  before(() => {
+    cy.visit(`${BASE_PATH}/app/home#/tutorial_directory/sampleData`);
+    // Click on "Sample data" tab
+    cy.contains('Sample data').click({ force: true });
```
Comment (CCongWang) [inline src/plugins/custom_import_map/cypress/integration/geojson_file_upload.spec.js:15]: could you try adding a timeout in this line? something like `cy.contains('Sample data, { timeout: 60000 }).click({ force: true });`

Comment (Shivamdhar) [reply src/plugins/custom_import_map/cypress/integration/geojson_file_upload.spec.js]: This worked like magic!

**Review by CCongWang — APPROVED** [review]

**Review by peterzhuamazon — COMMENTED** [review]
Thanks @Shivamdhar for making these changes and adding tests.

**Review by junqiu-lei — APPROVED** [review]

**Pull request merged**